### DISCUSSION TITLE
Sensoren aus EVCC für Home Assistant (Energie-Dashboard)

### DISCUSSION THREAD

**Discussion by 1977michael** [marq24/ha-evcc#38] [Q&A] [answered]
Hallo,

zunächst vielen Dank für die Integration, um EVCC in HA zu integrieren. Ich hatte auch das Problem, dass der WR (RCT) Fehlermeldungen sendet, da sowohl EVCC und HA auf ihn zugreifen...

Dank dieser Integration können ja sehr viele Daten, die EVCC sowieso ausliest, für HA verwendet werden, ohne sie manuell zu erstellen.

Nun wollte ich auch das Energie-Dashboard mit Sensoren "füttern", konnte aber nicht alle Werte finden.
Müssen die Energiewerte für PV und Batterie extra erstellt werden??

Vielen Dank für Eure Hilfe...

**Comment by marq24** [comment, marked as answer]
Das Energie Dashboard kann nur (k)Wh Sensoren verwenden - wenn Du also einen Sensor hast, der Dir nur (k)W liefert, dann musst/kannst Du mit Hilfe der 'Riemann Sum Integal Platform' (Einstellungen -> Geräte & Dienste -> Helfer -> Integralsensor) kannst Du einen solchen Zeitbasierten Sensor erzeugen

**Comment by 1977michael** [comment]
Hallo, vielen Dank für Deine Antwort. 
Das hat nun alles soweit geklappt. Über Template-Helfer und Integralsensoren sind nun alle Werte vorhanden...

Vielen Dank für Deine Arbeit!!
